plugins/woocommerce-blocks/tests/e2e/tests/products/products.block_theme.spec.ts

```diff
@@ -65,14 +66,12 @@ test.describe( `${ blockData.name } Block `, () => {
 			postId: 'woocommerce/woocommerce//archive-product',
 			postType: 'wp_template',
 		} );
-
-		await editor.canvas.locator( 'body' ).click();
-
+		await editor.enterEditMode();
+		await editor.setContent( '' );
```

Comment:
tip: Creating a new custom `archive-product` template would be more efficient as it wouldn't need to wait for the whole template to load before clearing it. Like in this test:

https://github.com/woocommerce/woocommerce/blob/460d73eee07e4ca28954f33d509d7c1414ad1f5b/plugins/woocommerce-blocks/tests/e2e/tests/add-to-cart-form/add-to-cart-form.block_theme.spec.ts#L85-L97

Reply:
👍 Addressed!

Reply:
I had to bring back the old logic as this specific test logic requires an actual product catalog template.

Reply:
I guess the `slug: 'product-catalog'` part should actually be `slug: 'archive-product'` so that it overwrites the original template and uses the correct logic. 🤔